e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js

```diff
@@ -131,15 +131,33 @@ describe('Mark as Unread', () => {
         // The New Messages line should appear above the selected post
         verifyPostNextToNewMessageSeparator('post2');
 
```

Comment:
I'm fine to modify this test case. With that, please change the title as follows.
```diff
- it('Should be able to mark channel as unread by alt-clicking', () => {
+ it('MM-T260 Mark as Unread New Messages line extra space moves with it', () => {

Reply:
Thank you, it's been updated.